lib/themes/dosomething/paraneue_dosomething/js/lab.js

```diff
@@ -15,10 +15,7 @@ $(document).ready(function() {
   if (Drupal.settings.dsOnboarding.enabled) {
     const jsxElementHook = 'jsx-onboarding';
     const slides = [ContextSlide, ContextSlide] // @TODO: create other slide components.
-    const campaign = {
-      // @TODO: switch to use Drupal.settings.dsCampaign once it is available.
-      title: 'Awesome Campaign Placeholder'
-    }
+    const campaign = Drupal.settings.dsCampaign || null;
```

Comment:
Not sure how others feel about this, but i generally prefer using `undefined` over `null` in :js: